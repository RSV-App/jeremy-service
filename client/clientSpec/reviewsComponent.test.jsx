import React from 'react';
import renderer from 'react-test-renderer';
import Reviews from '../src/components/ReviewsComponent.jsx';

describe('Reviews Component', () => {
  it('renders the reviews component and its children', () => {
    const tree = renderer
      .create(<Reviews restaurantId={1} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
