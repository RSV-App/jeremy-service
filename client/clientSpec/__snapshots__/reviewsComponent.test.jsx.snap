// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews Component renders the reviews component and its children 1`] = `
<div
  className="reviews"
>
  <div
    className="sorting-panel"
  >
    <h4>
      Sort by
    </h4>
    <div
      className="sort-area"
    >
      <div
        className="sort-dropdown"
        onClick={[Function]}
      >
        <span
          className="sorter-name"
        >
          Newest
        </span>
        <i
          className="sorter-icon"
        />
      </div>
      <div
        className="options-container"
      />
    </div>
    <h4>
      Filters
    </h4>
    <div
      className="filters"
    />
  </div>
</div>
`;
